reference/config_files/profiles.rst

```diff
@@ -520,6 +520,20 @@ Some of the capabilities of the profile templates are:
      [conf]
      tools.cmake.cmaketoolchain:toolchain_file = {{ os.path.join(profile_dir, "toolchain.cmake") }}
 
+- Getting settings from a filename, including referencing the current profile name. For
+  example, defining a generic profile which is configured according to its file name.
+  The variable ``profile_name`` pointing to the current profile file name is added to the context.
+
+  .. code-block:: jinja
+     :caption: *Linux-x86_64-gcc-12.jinja*
```

Comment:
```suggestion
     :caption: *Linux-x86_64-gcc-12*
```

Reply:
But why not drop the ``jinja`` extension?

Reply:
done, extension has been removed.